build.gradle

```diff
@@ -50,7 +50,7 @@ jar {
 }
 
 task extractGraphqlGrammar(type: Copy) {
-    from({ zipTree({ configurations.compile.files.find({ it.path.contains('/com.graphql-java/graphql-java/') }) }) }) {
+    from({ zipTree({ configurations.compile.files.find({ it.path.contains('com.graphql-java/graphql-java/') }) }()) }) {
```

Comment:
why changing that?

Reply:
When there is an issue (e.g. 'com.graphql-java/graphql-java/' is not found) the error message is very cryptic bc zipTree treats the argument as a closure (it says something like closure cannot be converted to path). Here we are executing the closure so that argument passed to zipTree will be always a path

Reply:
sounds good